Project tasks-Miloš Vojnović

milosvojnovic opened the pull request:
milos

Commit:
Project tasks-Miloš Vojnović

milos

MisaHack commented on src/MilošVojnović/Nasledjivanje.java:
```diff
@@ -5,7 +5,7 @@ interface Auto {
     }
-
+//promena
     interface Biciklo {
```
**Use English as a main language for programming.**
Rename each name that you use in the code from Serbian to English.
If you do not know English very good, **please use Google Translate to translate words - it is very important.**
**In professional programming you will always use English.**

Commit:
Project tasks-Miloš Vojnović

After a week of working in the IntelliJ program.
I crossed the basics of Java.
And this week I will try to upgrade my knowledge of Java.

MisaHack commented on src/MilošVojnović/ArithmeticOperations.java:
```diff
@@ -1,14 +1,14 @@
-public class AritmetickeOperacije {
+public class ArithmeticOperations {
     public static void main(String[] args) {
         float x = 10;// za precizne brojeve
```
// za precizne brojeve <---- comments should also be translated
